### ISSUE TITLE
upload example: failed to upload - pc 0.1.30

### ISSUE THREAD

**Issue by milochen0418** [pynecone-io/pynecone-examples#78]
```
My Testing Environment
        OS Darwin 22.4.0
        Pynecone 0.1.30
        Python 3.11.3
        Node v16.8.0
        Bun 0.5.9
```
The upload function is not working. 
When you click upload, the log is the following. 
```Task exception was never retrieved
future: <Task finished name='Task-61' coro=<AsyncServer._handle_event_internal() done, defined at /Users/milochen/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/socketio/asyncio_server.py:522> exception=ValidationError(model='Event', errors=[{'loc': ('payload',), 'msg': 'value is not a valid dict', 'type': 'type_error.dict'}])>
Traceback (most recent call last):
  File "/Users/milochen/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/socketio/asyncio_server.py", line 524, in _handle_event_internal
    r = await server._trigger_event(data[0], namespace, sid, *data[1:])
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/socketio/asyncio_server.py", line 569, in _trigger_event
    return await self.namespace_handlers[namespace].trigger_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/socketio/asyncio_namespace.py", line 37, in trigger_event
    ret = await handler(*args)
          ^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/pynecone-examples/lib/python3.11/site-packages/pynecone/app.py", line 594, in on_event
    event = Event.parse_raw(data)
            ^^^^^^^^^^^^^^^^^^^^^
  File "pydantic/main.py", line 549, in pydantic.main.BaseModel.parse_raw
  File "pydantic/main.py", line 526, in pydantic.main.BaseModel.parse_obj
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Event
payload
  value is not a valid dict (type=type_error.dict)
```

**Comment by milochen0418** [comment]
The PR in the `pynecone-io/pynecone.git` is related to this issue.
- https://github.com/pynecone-io/pynecone/pull/1042

**Comment by milochen0418** [comment]
After testing, the bug if solved in `pynecone==0.1.31`
